services/backend/services/v4/compose.base.yaml

```diff
@@ -7,6 +7,12 @@ services:
         condition: service_started
     volumes:
       - ./config/functionalAccounts.json:/home/node/app/functionalAccounts.json
+      - ../../../../entrypoints/loop_entrypoints.sh:/usr/local/bin/loop_entrypoints.sh
```

Comment:
could you please use `${PWD}` instead of ../../../? I think it improves readability